compatible/pom.xml

```
@@ -87,11 +87,23 @@
             <version>2.1.0-SNAPSHOT</version>
             <scope>compile</scope>
         </dependency>
-              <dependency>
+        <dependency>
             <groupId>org.apache.seata</groupId>
             <artifactId>seata-integration-tx-api</artifactId>
             <version>2.1.0-SNAPSHOT</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.seata</groupId>
+            <artifactId>seata-tcc</artifactId>
+            <version>2.1.0-SNAPSHOT</version>
```

```suggestion
            <version>${project.version}</version>
```
Why not use ${project.version}?

---

Done.